src/spikeinterface/sorters/basesorter.py

```diff
@@ -169,14 +169,12 @@ def params_description(cls):
     @classmethod
     def set_params_to_folder(cls, recording, output_folder, new_params, verbose):
         params = cls.default_params()
+        valid_parameters = cls.params_description().keys()
```

Comment:
You've already got the params above. Why not just make valid params from

params.keys()

Wouldn't that be clearer than pulling in the descriptions dict and using those keys?

Reply:
Yes. Agree. Changing.

Reply:
Done.